src/esp/metadata/attributes/SceneDatasetAttributes.cpp

```diff
@@ -25,8 +25,7 @@ SceneDatasetAttributes::SceneDatasetAttributes(
 bool SceneDatasetAttributes::addNewSceneInstanceToDataset(
     const attributes::SceneAttributes::ptr& sceneInstance) {
   // info display message prefix
-  const std::string infoPrefix("::addNewSceneInstanceToDataset : Dataset : '" +
-                               getSimplifiedHandle() + "' : ");
+  const std::string infoPrefix("Dataset : '" + getSimplifiedHandle() + "' : ");
```

Comment:
Nit: This is going to require 3 reallocations of the prefix in the constructor of the String. Should all be std::string